sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_operations_mixin.py

```diff
@@ -13,8 +13,7 @@
 import warnings
 
 # FIXME: have to manually reconfigure import path for multiapi operation mixin
-from .._lro import AnalyzeHealthcareEntitiesLROPoller, AnalyzeHealthcareEntitiesLROPollingMethod
-from .._lro import AnalyzeBatchActionsLROPoller, AnalyzeBatchActionsLROPollingMethod
+from .._lro import AnalyzeBatchActionsLROPoller, AnalyzeBatchActionsLROPollingMethod, AnalyzeHealthcareEntitiesLROPoller, AnalyzeHealthcareEntitiesLROPollingMethod
```

Comment:
I feel like I'm going crazy, because current test failures are showing this import with four dots instead of 2...

Reply:
Ok, this is maybe mysteriously fixed now, as I'm seeing new problems.  :)